images/prometheus-blackbox-exporter/tests/main.tf

```diff
@@ -0,0 +1,64 @@
+terraform {
+  required_providers {
+    oci       = { source = "chainguard-dev/oci" }
+    imagetest = { source = "chainguard-dev/imagetest" }
+  }
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+variable "chart_version" {
+  description = "The version of the chart to test."
+  default     = "latest"
+}
+
+data "imagetest_inventory" "this" {}
+
+resource "imagetest_harness_k3s" "this" {
+  name      = "prometheus-blackbox-exporter"
+  inventory = data.imagetest_inventory.this
+}
+
+resource "random_pet" "suffix" {}
+
+module "helm" {
+  source = "../../../tflib/imagetest/helm"
+
+  name          = "prometheus-blackbox-exporter-${random_pet.suffix.id}"
+  namespace     = "prometheus-blackbox-exporter-${random_pet.suffix.id}"
+  repo          = "https://prometheus-community.github.io/helm-charts"
+  chart         = "prometheus-blackbox-exporter"
+  chart_version = var.chart_version
+
+  values = {
+    image = {
+      registry   = data.oci_string.ref.registry
+      repository = data.oci_string.ref.repo
+      tag        = data.oci_string.ref.pseudo_tag
+    }
+  }
+}
+
+resource "imagetest_feature" "basic" {
```

Comment:
We'll need to expand the testing here - a helm install alone won't catch much. Can we add a shell test which validates functionality? The [upstream README](https://github.com/prometheus/blackbox_exporter) seems to give a good example to check its working, so we'd do that in shell, and some response / log validation.